src/test/e2e-emulated/contracts/send.tact

```diff
@@ -10,4 +10,43 @@ contract SendTester with Deployable {
         self.reply("World".asComment());
         emit("Something".asComment());
     }
-}
\ No newline at end of file
+
+    /// A no-op receiver for incoming funds
+    receive("topup") {}
+
+    /// A getter to get contracts balance
+    get fun balance(): Int { return myBalance(); }
+
+    /// Sends a message, keeps the original balance, sends the rest in the next message
+    receive("ReserveAtMost_1") {
+        emit("Have you seen this message?".asComment());
+
+        nativeReserve(ton("0.05"), ReserveAtMost | ReserveAddOriginalBalance);
+
+        send(SendParameters{
+            to: sender(),
+            value: 0,
+            mode: SendRemainingBalance,
+            body: "I give you my all! Well, all that's not mine!".asComment(),
+        });
+    }
+
+    /// Sends a message, keeps the original balance, sends the rest in the next message
```

Comment:
The same comment as for `receive("ReserveAtMost_1")`. Let's clarify the difference